data/json/monsters/zed_misc.json

```
@@ -836,6 +836,53 @@
     "upgrades": { "half_life": 20, "into": "mon_zombie_hunter" },
     "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "PUSH_MON", "FILTHY" ]
   },
+  {
+    "id": "mon_zombie_regenerate",
+    "type": "MONSTER",
+    "name": { "str": "regenerating zombie" },
+    "description": "A hairless zombie; its pale pinkish flesh appears to be squirming and moving across its body, quickly covering and fixing any wounds that it receives.  Its clothing is riddled with holes, suggesting someone else tried to take it down before you.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
```

```suggestion
    "hp": 60,
```
It's gonna be tough enough to kill this thing, I think the base level zombie should be pretty fragile. We can boost this as it evolves... this sets it somewhat lower than a fast zombie, but if you're not doing pretty decent damage against it, it willl have more staying power than a normal.